src/components/ui/time-picker.tsx

```diff
@@ -93,7 +93,7 @@ const TimePickerInput = React.forwardRef<HTMLInputElement, TimePickerInputProps>
         setDate(setDateByType(tempDate, newValue, picker, period));
       }
       if (e.key >= '0' && e.key <= '9') {
-        if (picker === '12hours') setPrevIntKey(e.key);
+        if (picker === 'hours') setPrevIntKey(e.key);
```

Comment:
what are possible values of picker other than hours

Reply:
Previously there are 2 options like 12hours format or 24hours format.

I removed 24hours related code as it was redundant.